src/BlockNotice/index.tsx

```diff
@@ -0,0 +1,17 @@
+import "./styles.scss";
+
+type BlockNoticeProps = {
+    title: string,
+    description: string,
+    href: string
+}
+
+export default function BlockNotice({title, description, href}: BlockNoticeProps) {
```

Comment:
I fully understand the intention of this component, and this is great. Just what we need. But, what do you think of the idea of having an option to add custom stuff in this component using the `children` prop?
Someday we might want to add buttons or more than one link. Also, maybe we can reuse this component to show different types of notices (success, info, error, etc..)

Reply:
I think adding children is a good idea. I can see different notice types being helpful as well but it would be something we need to include @jdghinson on so I'd like to come back and readdress that if we want to go that route. For the children prop are you thinking something as simple as including it at the bottom? 591577ca7f0abdf89f8512c612d437b69f5ded1c  It also seems this copy was missing the anchor text so I had included it a4d0c9563c0a80becbeab0e30a86b79780f91bba

Reply:
Yeah I agree, we should ask Jeffrey what he thinks. 
If we go with having the `children` prop route, then it would be a good idea to make the `description` and `href` props optional.  In other words, if I don't provide a description or href prop, they won't show up. Regarding the `children` placement, I think having it above the link might be a better option, but I'll leave that up to you.

Reply:
Actually your right, above the anchor makes it more extendible. I'll bring the notice types up to Jeffrey when we review the rest of the block requirements. ca33d8bb702d9ae2f2bcacfcfbf9fc0eaad5040c. Thanks!